cosim/emulator/src/dpi.cc

```diff
@@ -0,0 +1,115 @@
+#ifdef COSIM_VERILATOR
+#include <VTestBench__Dpi.h>
+#endif
+
+#include <csignal>
+
+#include <glog/logging.h>
+#include <fmt/core.h>
+
+#include "svdpi.h"
+#include "vbridge_impl.h"
+#include "exceptions.h"
+
+static bool terminated = false;
+
+
+void sigint_handler(int s) {
+  terminated = true;
+  dpiFinish();
+}
+
+
+#define TRY(action) \
```

Comment:
#define TRY(action) I guess the following "action" -s might be better to be "(action)" -s.

Reply:
cc @SharzyL

Reply:
`action` is a expected to be a statement or braced compound statements. Parenthesis is not required since it is enclosed in a brace.